tests/Sharpliner.Tests/AzureDevOps/ConditionedExpressions/ConditionalsTests.cs

```diff
@@ -31,7 +31,7 @@ public void And_Condition_Test()
         variable.Condition!.ToString().Should().Be(
             "and(" +
                 "notIn('bar', 'foo', 'xyz', 'foo'), " +
-                "ne($(Configuration), 'Debug'), " +
+                "ne('$(Configuration)', 'Debug'), " +
```

Comment:
These tests were asserting an incorrect syntax. I made EqualityCondition inherit from StringCondition and it fixed this incorrect syntax, so I've updated all these tests to work now.

Reply:
I can see there are some failing around quoting `boolean` values. Do you know what's the correct syntax in YAML? Does it matter if you specify variable is `true` and compare with `'true'`?

Reply:
Thanks for trying it out!